Epsilon/Components/KpiMatrixComponent.cs

```diff
@@ -0,0 +1,208 @@
+using DocumentFormat.OpenXml;
+using DocumentFormat.OpenXml.Packaging;
+using DocumentFormat.OpenXml.Wordprocessing;
+using Epsilon.Abstractions;
+using Epsilon.Abstractions.Components;
+
+namespace Epsilon.Components;
+
+public class KpiMatrixComponent : AbstractCompetenceComponent
+{
+
+    public KpiMatrixComponent(IAsyncEnumerable<LearningDomainSubmission> submissions, IEnumerable<LearningDomain?> domains, IEnumerable<LearningDomainOutcome> outcomes)
+        : base(submissions, domains, outcomes)
+    {
+    }
+
+    public override async Task<Body?> AddToWordDocument(MainDocumentPart mainDocumentPart)
+    {
+         var body = mainDocumentPart.Document.Body;
+        // Create a table, with rows for the outcomes and columns for the assignments.
```

Comment:
Have the same check for null of body as in ComptenceProfileComponent or use question marks. One of the two